File: app/src/main/java/feelings/guide/util/ToastUtil.java
package feelings.guide.util;

import android.content.Context;
import android.view.Gravity;
import android.widget.Toast;

public class ToastUtil {

    public static void showLong(final Context context, final String message) {
        Toast.makeText(context, message, Toast.LENGTH_LONG).show();
    }

    public static void showShort(Context context, String message) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
    }

    /**
     * Needed on edit screens, where the keyboard is being rendered, in order not to overlap with the keyboard.
     */
    public static void showShortTop(final Context context, final String message) {
        Toast toast = Toast.makeText(context, message, Toast.LENGTH_SHORT);
        toast.setGravity(Gravity.TOP, 0, 200);
        toast.show();
    }
}

File: app/src/main/kotlin/feelings/guide/util/ToastUtil.kt
package feelings.guide.util

import android.content.Context
import android.view.Gravity
import android.widget.Toast

object ToastUtil {

    fun showLong(context: Context, message: String) {
        val toast = Toast.makeText(context, message, Toast.LENGTH_LONG)
        toast.setGravity(Gravity.BOTTOM, 0, 100)
        toast.show()
    }

    fun showShort(context: Context, message: String) {
        val toast = Toast.makeText(context, message, Toast.LENGTH_SHORT)
        toast.setGravity(Gravity.BOTTOM, 0, 100)
        toast.show()
    }

    /**
     * Needed on edit screens, where the keyboard is being rendered, in order not to overlap with the keyboard.
     */
    fun showShortTop(context: Context, message: String) {
        val toast = Toast.makeText(context, message, Toast.LENGTH_SHORT)
        toast.setGravity(Gravity.TOP, 0, 200)
        toast.show()
    }
}